modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/notifications/ScheduledPublicationUserNotificationHandler.java

```diff
@@ -42,10 +42,28 @@ protected String getBody(
 		JSONObject jsonObject = _jsonFactory.createJSONObject(
 			userNotificationEvent.getPayload());
 
-		boolean showConflicts = jsonObject.getBoolean("showConflicts");
+		boolean admin = jsonObject.getBoolean("isAdmin");
```

Comment:
We don't need a new flag variable. You can use `showConflicts` to determine the cause of the failure. If `showConflicts` is false, that means the failure is due to java exceptions.

Reply:
One of the notifications caused by the conflicts should be sent only to the system admin, so we need to check  that it is being sent directly to them